src/main/app/forms/models/multiRowFormItem.ts

```diff
@@ -5,4 +5,8 @@ export abstract class MultiRowFormItem {
   isEmpty (): boolean {
     return Object.values(this).every(value => !value)
   }
+
+  isAtLeastOneFieldPopulated (): boolean {
```

Comment:
minor naming suggestion: `isAnyFieldPopulated`

EDIT: Now that I think about it, shouldn't it be an exact opposite of `isEmpty`? And given that, why not just use `!row.isEmpty()` everywhere instead of this?